src/components/SignUp/SignUp.tsx

```diff
@@ -3,18 +3,40 @@ import EmailPasswordForm from '../EmailPasswordForm'
 import FormLinks from '../FormLinks'
 import './SignUp.css'
 
-export interface Props {
+/**
+ * @typedef ISignUpProps
+ * @props {string} [className] - the CSS classes
+ * @props {() => void} [onClick] - onClick handler launching after submit form
+ * @props {unknown} [apiError] - api error messages
+ */
+
+export interface ISignUpProps {
   className?: string
   onClick?: () => void
   apiError?: unknown
 }
 
+/**
+ * Renders the sign-up component
+ *
+ * @param  {string} [classname] - the CSS classes
+ * @param  {() => void} [onClick] - onClick handler launching after submit form
+ * @param  {unknown} [apiError] - api error messages
```

Comment:
We should avoid using `unknown` where it's possible. Because we want to display the `apiError`, then we need a string. So let's change this to the string and then the `SignUpContainer` will convert the error object into string. Like this:

**In SignUpContainer...**
1. API returns some error, ie.:

```
// This is only an example:
{ 
 status: 400,
 code: 400,
 message: 'Something went wrong'
}
```

2. Extract something from the response, ie. `error.message`

3. Pass the the `error.message (as string)` to the `SignUpForm`

**In SingUpForm...**

4. If `apiError` exists, render it.